cob_hardware_emulation/scripts/emulation_move_base.py

```diff
@@ -0,0 +1,122 @@
+#!/usr/bin/env python
+
+import actionlib
+import copy
+import math
+import rospy
+import tf
+
+from geometry_msgs.msg import Twist
+from move_base_msgs.msg import MoveBaseAction, MoveBaseActionResult
+from nav_msgs.msg import Odometry
+
+from sensor_msgs.msg import JointState
+
+class emulation():
```

Comment:
specific names, e.g. `MoveBaseEmulation` (and `FollowJointTrajectoryEmulation` respectively)